frontend/src/component/admin/banners/BannerModal/BannerForm.tsx

```diff
@@ -10,7 +10,7 @@ import { ChangeEvent, Dispatch, SetStateAction, useState } from 'react';
 import { Visibility } from '@mui/icons-material';
 import { BannerDialog } from 'component/banners/Banner/BannerDialog/BannerDialog';
 
-const StyledForm = styled('form')(({ theme }) => ({
+const StyledForm = styled('div')(({ theme }) => ({
```

Comment:
Scouting: This meant that a `form` would be inside another `form`, which caused a console error.